.github/workflows/db-charm-tests.yaml

```diff
@@ -8,19 +8,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        charm-repo:
-          - "canonical/postgresql-operator"
-          - "canonical/postgresql-k8s-operator"
-          - "canonical/mysql-operator"
-# TODO: uncomment once secrets issues are fixed in this charm:
-#       https://github.com/canonical/mysql-k8s-operator/pull/371
-#          - "canonical/mysql-k8s-operator"
+        include:
+          - charm-repo: canonical/postgresql-operator
+            tag: rev405
```

Comment:
It seems like there are some commits (e.g. 82e6cb91cae5123e86ffc24d24a1e1a2fc1f8043) that don't get tags, maybe ones done by bots? Wouldn't it be better to just go with whatever the latest hash is?

Reply:
I've changed pins back to commits